src/test/java/org/prebid/server/bidder/colossus/ColossusBidderTest.java

```diff
@@ -75,7 +77,7 @@ public void makeHttpRequestsShouldReturnExpectedBidRequest() {
 
         // then
         final BidRequest expectedRequest = bidRequest.toBuilder()
-                .imp(singletonList(bidRequest.getImp().get(0).toBuilder().tagid("tagidString").build()))
+                .imp(singletonList(bidRequest.getImp().get(0).toBuilder().tagid("tagidStr").build()))
```

Comment:
Why do we need this change?
To be fair, I don't like neither initial name, nor changed one :)